test/image/mocks/gl3d_tetrahedra.json

```diff
@@ -16,6 +16,8 @@
         [1, "rgb(0, 0, 255)"]
       ],
       "intensity": [0, 0.33, 0.66, 1],
+      "cmin": -0.5,
+      "cmax": 0.5,
```

Comment:
fixed in f55c6f5dad3c302e265a246d47c2b15d563daa52 via https://github.com/gl-vis/gl-error3d/pull/3